content/id/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Kontroler <i>Ingress</i>
+title: Kontroler *Ingress*
```

Comment:
nit: You most likely can omit the emphasis of the `title` field of the front matter.

Reply:
yes I think title doesn't support it as well

![image](https://user-images.githubusercontent.com/3192347/56580888-5e972c00-65f1-11e9-9fb6-b13e47993574.png)

Reply:
Right, 👍